### PR TITLE
Translated to sr/me

### PR DESCRIPTION
**Pull request by vido89**
I'v just translated this to ME/Serbian

EDIT: Reference #254 .

### CODE REVIEW

File: po/sr_ME@latin.po [hunk 1]
```diff
@@ -0,0 +1,566 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the PACKAGE package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
+#
```
Comment (12345ieee) [inline po/sr_ME@latin.po]: The initial section has to be filled in, with the standard GC header (look in the po folder) and your contact info (for attribution).

File: po/sr_ME@latin.po [hunk 2]
```diff
@@ -0,0 +1,566 @@
...
+"%10<=4 && (n%100<10 || n%100>=20) ? 1 : 2);\n"
+"Language: sr_ME@latin\n"
+
+#: ../gui/GameConqueror.ui.h:1
+msgid "GameConqueror - Memory Editor"
+msgstr "GameConqueror - Urednik Memorije"
+
+#: ../gui/GameConqueror.ui.h:2
+msgid "_Address:"
+msgstr "_Adresa"
```
Comment (12345ieee) [inline po/sr_ME@latin.po]: Missing ':'

File: po/sr_ME@latin.po [hunk 3]
```diff
@@ -0,0 +1,566 @@
...
+#: ../gui/GameConqueror.ui.h:48
+msgid "Data _Type:"
+msgstr "Tip _podatka:"
+
+#: ../gui/GameConqueror.ui.h:49
+msgid ""
+"<b>Basic:</b>      Fastest but may miss some values\n"
+"<b>Normal:</b>  Works for most cases\n"
+"<b>Full:</b>         Never miss values but slowest"
+msgstr ""
```
Comment (12345ieee) [inline po/sr_ME@latin.po]: Mismatched a translation here.

File: po/sr_ME@latin.po [hunk 4]
```diff
@@ -0,0 +1,566 @@
...
+" For bytearray: use 2-char hexadecimal notation for each byte, separated by "
+"spaces\n"
+" e.g. <span font_family=\"monospace\">FE DC BA 98 76 54 32 10</span>\n"
+" <span font_family=\"monospace\">??</span> can be used as a wildcard value\n"
+"\n"
+" For string: enter the string directly"
+
+#: ../gui/GameConqueror.ui.h:52
+msgid "_Search Scope:"
+msgstr "_pretraga:"
```
Comment (12345ieee) [inline po/sr_ME@latin.po]: No capital letter?

File: po/sr_ME@latin.po [hunk 5]
```diff
@@ -0,0 +1,566 @@
...
+msgid "_Close"
+msgstr "_Zatvori"
+
+#: ../gui/GameConqueror.ui.h:69
+msgid "Memory address of the value"
+msgstr "Memorijska adresa vrijednosti"
+
+#: ../gui/GameConqueror.ui.h:70
+msgid "Addres_s:"
+msgstr "Adresa:"
```
Comment (12345ieee) [inline po/sr_ME@latin.po]: Missing '_'

File: po/sr_ME@latin.po [hunk 6]
```diff
@@ -0,0 +1,566 @@
...
+msgid "Report bugs to "
+msgstr "Prijavi grešku"
+
+#: ../gui/GameConqueror.py:1187
+msgid "prefill the search box"
+msgstr "Popunite polje za pretragu"
+
+#: ../gui/GameConqueror.py:1189
+msgid "PID of the process"
+msgstr "Redni broj procesa"
```
Comment (12345ieee) [inline po/sr_ME@latin.po]: `PID` is a technical term, is this correct?

Comment (vido89) [inline po/sr_ME@latin.po]: Yes its correct, I assume that PID is abbreviation for Process ID

File: po/sr_ME@latin.po (hunk 1 above)
Comment (vido89) [inline po/sr_ME@latin.po]: PoEdit does not show header stuff so I edited it manualy

File: po/sr_ME@latin.po (hunk 2 above)
Comment (vido89) [inline po/sr_ME@latin.po]: Fixed

File: po/sr_ME@latin.po (hunk 3 above)
Comment (vido89) [inline po/sr_ME@latin.po]: I dont know how this happened, its ok now

File: po/sr_ME@latin.po (hunk 4 above)
Comment (vido89) [inline po/sr_ME@latin.po]: Yup you are right

File: po/sr_ME@latin.po (hunk 5 above)
Comment (vido89) [inline po/sr_ME@latin.po]: Im not sure for what "_s" stands for ?

**Comment by vido89** [comment]
Should I close this PR and create new since I dont know how to update it ?

**Comment by 12345ieee** [comment]
Looks updated to me, @vido89 did you have any issue?

**Comment by vido89** [comment]
Ok great I just made new PR its automatically updated I guess

**Comment by 12345ieee** [comment]
Ok, the file looks good (I don't need the mo, but I can delete it myself).

Just to be sure, is the locale code correct? I can't see it on my system.
Aka, do the other translations reside in `/usr/share/locale/sr_ME@latin`?

**Comment by vido89** [comment]
I also do not have /usr/share/locale/sr_ME@latin but local code is correct the language is Serbian/Montenegrian version of it.Poedit automatically created sr_ME@latin for me when I entered Montenegrian language its fine, language is new, check it on google :)

**Comment by 12345ieee** [comment]
Oh, well, I updated the relative files and pushed the commit at https://github.com/scanmem/scanmem/tree/sr_me_translation .

Build GC from there, install it and check that your locale works, as I cannot test it.

Thanks for contributing.

**Comment by vido89** [comment]
Ok I just compiled it and installed it but I cant find a way to change language, how do I do that ?

**Comment by 12345ieee** [comment]
Linux i18n does not usually work that way, the program language is controlled by the `LANG` & friends envvars.
Then you need to have the language messages installed to `/usr/share/locale/<CORRECT LOCALE NAME>/LC_MESSAGES/GameConqueror.mo`

In my case, language is Italian and GC install setup correctly installs `/usr/share/locale/it/LC_MESSAGES/GameConqueror.mo`.

You need to run `echo $LANG` to check which locale are you using and figure out the right language code we need to call the po file.

**Comment by vido89** [comment]
When I run `echo $LANG` I get `en_US.UTF-8` but I cant chage it in region and langugage strange ?

**Comment by 12345ieee** [comment]
Looks like your language code is just `sr_ME`, without `@latin`.

I was able to install the locale support via `sudo dpkg-reconfigure locales` and then `LANG=sr_ME.UTF-8 python gui/GameConqueror.py` worked.

You have to look up how to set the locale in your desktop environment, I can't help you there, but I can confirm it works on my end.

I'll commit this, thanks for the translation.

**Comment by 12345ieee** [comment]
Merged in 9b8e0b9 , thanks.

**Comment by vido89** [comment]
You're welcome

**Pull request closed without merging**